feature-libs/organization/unit-order/styles/_order-history.scss

```diff
@@ -83,8 +83,8 @@
   }
```

Comment:
Naming could be improved, even though it's not easy to do. Maybe the only name I'd like to see changed is the name of this file, back to how it was, because currently we have two files named `_order-history.scss`.

Reply:
thanks i missed that